component/src/main/ballerina/redis/actions.bal

```diff
@@ -14,1030 +14,816 @@
-
-        P{{key}} The key
-        P{{value}} The value to be set
-        P{{offset}} The offset at which the value should be set
-        R{{}} The original bit value stored at offset or `error` if an error occurs
-    }
+    # Sets or clears the bit at offset in the string value stored at key.
+    #
+    # + key - The key
+    # + value - The value to be set
+    # + offset - The offset at which the value should be set
+    # + return - The original bit value stored at offset or `error` if an error occurs
     public extern function setBit(string key, int value, int offset) returns (int|error);
 
-    documentation {
-        Set the value and expiration of a key.
-
-        P{{key}} The key
-        P{{value}} The value to be set
-        P{{expirationPeriodSeconds}} Expiration time to be set, in seconds
-        R{{}} New value of the key or `error` if an error occurs
-    }
+    # Set the value and expiration of a key.
+    #
+    # + key - The key
+    # + value - The value to be set
+    # + expirationPeriodSeconds - Expiration time to be set, in seconds
+    # + return - New value of the key or `error` if an error occurs
     public extern function setEx(string key, string value, int expirationPeriodSeconds)
         returns (string|error);
 
-    documentation {
-        Set the value of a key, only if the key does not exist.
-
-        P{{key}} The key
-        P{{value}} The value to be set
-        R{{}} New value of the key or `error` if an error occurs
-    }
+    # Set the value of a key, only if the key does not exist.
+    #
+    # + key - The key
+    # + value - The value to be set
+    # + return - New value of the key or `error` if an error occurs
     public extern function setNx(string key, string value) returns (boolean|error);
 
-    documentation {
-        Overwrite part of a string at key starting at the specified offset.
-
-        P{{key}} The key
-        P{{offset}} The offset at which the value should be set
-        P{{value}} The value to be set
-        R{{}} The length of the string after it was modified or `error` if an error occurs
-    }
+    # Overwrite part of a string at key starting at the specified offset.
+    #
+    # + key - The key
+    # + offset - The offset at which the value should be set
+    # + value - The value to be set
+    # + return - The length of the string after it was modified or `error` if an error occurs
     public extern function setRange(string key, int offset, string value)
         returns (int|error);
 
-    documentation {
-        Get the length of the value stored in a key.
-
-        P{{key}} The key
-        R{{}} the length of the string at key, or 0 when key does not exist or `error` if an error occurs
-    }
+    # Get the length of the value stored in a key.
+    #
+    # + key - The key
+    # + return - the length of the string at key, or 0 when key does not exist or `error` if an error occurs
```

Comment:
Capitalize the first letter.

Reply:
Fixed with https://github.com/wso2-ballerina/package-redis/pull/28